src/Connection/Connection.php

```diff
@@ -6,6 +6,9 @@
 
 class Connection
 {
+    /**
+     * @param non-empty-string $address
+     */
```

Comment:
```suggestion
    /**
     * @param non-empty-string $address
     *
     * ```php
     * new Connection('localhost:7233');
     * ```
     */
```

Examples are preferred